javascript/src/HtmlEditorField.js

```diff
@@ -844,7 +844,14 @@ $.entwine('ss', function($) {
 				action = "update";
 			}
 
-			if(href.match(/^mailto:(.*)$/)) {
+            if(href.match(/^mailto:(.*)\?subject=(.*)$/)) {
```

Comment:
The link editor UI doesn't have a way to create a link like that. If the user manually creates a link with a body then they're going to have manually edit it in the future too. Or a you suggesting we add a field for editing the body in the link editor too?